### DISCUSSION TITLE
钱去哪了？（我们6个钱包以及三十年房贷）

### DISCUSSION THREAD

**Discussion by frshgrass** [WeNeedHome/SummaryOfLoanSuspension#1008] [General]
事实就是：我们30年的贷款，以及现在地方政府的天量地方债，七成左右变成了各种三四线城市的各种高楼和云贵各种高速大桥，两成左右被地方实权领导及其家属贪污，其他一成就是各种开发商和被层层七包八包的包工头拿掉。

最后各位还需要还上百万亿的地方债，这是通过挤压福利（医保社保）加上通货膨胀隐性挤压各位福利和洗劫各位存款。

然后在京东淘宝拼多多，各种创始人高位血亏套现，各种red二代抢班夺权吃相难看（"拿来吧！你！"），真的一个字 恶心！！！

**Comment by winnoryder** [comment]
买房后，老百姓破产了，地产商也破产了，钱都去哪了？

**Reply by wedxz** [reply]
年年维稳支出比军费还高，官老爷二奶都六个了

**Reply by volans19** [reply]
不愁没地去，一个官老爷就几十上百的私生子，养他们上学炫富不得有开销？2345678奶，一人几十辆跑车几十套豪宅不得有开销？自己在墙内洗脑老百姓人人喊打的日本，美国，英法德，一买几百套房，不得有开销？墙内几百个人一年的工资加一块都不够人家一个保姆吃一个月的

**Comment by zxcs001** [comment]
共产党红色血脉把老百姓当韭菜往四里面割，让全民背上几十年负债，然后把资产全部放到瑞士银行呗。

中国人（99%， 包括什么体制内的，本质上都是些打手给红色血脉干活的狗罢了）都是些牛马，牲畜，作为人唯一的价值就是提供生产力，为了红色贵族积累在国际上能够交换的财富罢了。你拿的那些红票票都是些代金券而已，为什么正儿八经的红色血脉都不屑这些东西？因为他们知道，要真是印钞票的话，他们想印多少就印多少，这玩意可以分分钟变的一分钱都不值。所以正儿八经的常委之类的，都是直接持有资产和美金。这些玩意才是实打实有价值的东西。

所以看清楚这一点的人（不是有钱人就是高知、体制内中高层），早就有打算了，不过大多数人还是看不清形势。

你没看这些天，Youtube上那些国人失业的，贷款还不上家破人亡的视频。大家基本上都是在看笑话的心态来看的。

看中国人被铁拳砸的生不如死，成了不少海外网民每日娱乐的一部分了……
